avm/ptn/data/private-analytical-workspace/tests/common.tests.ps1

```diff
@@ -0,0 +1,393 @@
+function Test-VerifyLock($ResourceId) {
```

Comment:
I'm not sure this file will be called by the CI in its current location. If you want post-deployment tests, they should be in the same folder (or sub-folder) of the test the post-deployment test should run for